Frameworks/CoreFoundation/Preferences.subproj/CFPreferences.c

```diff
@@ -882,11 +882,7 @@ static Boolean synchronizeApplicationDomain(CFTypeRef context, void *domain) {
     CFDataRef data = CFPropertyListCreateData(__CFPreferencesAllocator(), (CFPropertyListRef)domain, kCFPropertyListXMLFormat_v1_0, 0, NULL);
     Boolean success = false;
     if (data) {
-#if DEPLOYMENT_TARGET_WINDOWS
-        success = _CFWriteBytesToFileAsync((CFURLRef)context, CFDataGetBytePtr(data), CFDataGetLength(data));
```

Comment:
That seems pretty unlikely? I would guess that the majority of time spent is in the actual file write. If a user finds that time to be too long, they can always offload to a background thread themselves, now that synchronize no longer has a requirement that it be called on the main thread.